docs/2-advanced/18-custom-elements-manifest.md

```diff
@@ -0,0 +1,313 @@
+# Custom Elements Manifest
+
+This article describes the prerequisites for **Custom Elements Manifest** generation and how the custom analyzer works for UI5 Web Components project and third-party projects.
```

Comment:
we should add a few words what the custom element manifest is, and that it is used for creating wrappers (react/angular/ui5). a mention of the custom-elements-internal.json would be also good.